dataplane/standalone/apigen/xml.go

```diff
@@ -48,18 +48,20 @@ type MemberDef struct {
 
 // EnumValue is a single values in a enum.
 type EnumValue struct {
-	Name                string              `xml:"name"`
-	Initializer         string              `xml:"initializer"`
-	DetailedDescription DetailedDescription `xml:"detaileddescription"`
+	Name                string      `xml:"name"`
+	Initializer         string      `xml:"initializer"`
+	DetailedDescription Description `xml:"detaileddescription"`
+	BriefDescription    Description `xml:"briefdescription"`
 }
 
-// DetailedDescription contains extra information about an enum value.
-type DetailedDescription struct {
+// Description contains extra information about an enum value.
+type Description struct {
 	Paragraph Paragraph `xml:"para"`
 }
 
 // Paragraph is a generic paragraph.
 type Paragraph struct {
+	InlineText string       `xml:",chardata"` // For BriefDescription, the paragraph contains raw text.
```

Comment:
Does DetailedDescription also have this field? From reading it it seems it may deserve a separate Paragraph struct.

Reply:
yeah, they both have it sometimes